evm/src/fuzz/strategies/invariants.rs

```diff
@@ -86,9 +88,25 @@ fn generate_call(
 /// * If `senders` is empty, then it's a completely random address.
 /// * If `senders` is not empty, then there's an 80% chance that one from the list is selected. The
 ///   remaining 20% will be random.
-fn select_random_sender(senders: Vec<Address>) -> impl Strategy<Value = Address> {
-    let fuzz_strategy =
-        fuzz_param(&ParamType::Address).prop_map(move |addr| addr.into_address().unwrap()).boxed();
+fn select_random_sender(
+    fuzz_state: EvmFuzzState,
+    senders: Vec<Address>,
+) -> impl Strategy<Value = Address> {
+    let fuzz_strategy = proptest::strategy::Union::new_weighted(vec![
+        (
+            10,
+            fuzz_param(&ParamType::Address)
+                .prop_map(move |addr| addr.into_address().unwrap())
+                .boxed(),
+        ),
+        (
+            90,
+            fuzz_param_from_state(&ParamType::Address, fuzz_state)
```

Comment:
Looks like weights are 90/10 but the comment above this method says 80/20. Probably should be exposed as a config option at some point (not necessarily in this PR)